src/utils/twoFactor.js

```diff
@@ -4,6 +4,8 @@ import { WalletError } from './walletError'
 import { promptTwoFactor } from '../actions/account'
 import { ACCESS_KEY_FUNDING_AMOUNT, convertPKForContract, toPK } from './wallet'
 
+window.nearApiJs = nearApiJs
```

Comment:
@mattlockyer why these `window.` assignments are needed? Is this debug leftover?

Reply:
Yes sorry, will remove and merge. Thanks!